integration/js/AudioTest.js

```diff
@@ -1,56 +1,89 @@
     const useStereoMusicAudioProfile = this.payload.useStereoMusicAudioProfile ? this.payload.useStereoMusicAudioProfile : false;
 
-    await WaitForMeetingToBeCreated.executeStep(this, session);
+    await test_window.runCommands(async () => await this.addUserToMeeting(test_attendee_id, session, useStereoMusicAudioProfile));
+    await monitor_window.runCommands(async () => await this.addUserToMeeting(monitor_attendee_id, session, useStereoMusicAudioProfile));
+
+    await monitor_window.runCommands(async () => await RosterCheck.executeStep(this, session, 2));
+    await test_window.runCommands(async () => await RosterCheck.executeStep(this, session, 2));
+
+    await test_window.runCommands(async () => await PlayRandomToneStep.executeStep(this, session, generateStereoTones));
+    await monitor_window.runCommands(async () => await RemoteAudioCheck.executeStep(this, session, "AUDIO_ON", generateStereoTones));
+
+    await test_window.runCommands(async () => await ClickMicrophoneButton.executeStep(this, session, 'OFF'));
+    await monitor_window.runCommands(async () => await RemoteAudioCheck.executeStep(this, session, "AUDIO_OFF", generateStereoTones));
+  }
+
+  async addUserToMeeting(attendeeId, session, useStereoMusicAudioProfile) {
     await OpenAppStep.executeStep(this, session);
-    await AuthenticateUserStep.executeStep(this, session, this.attendeeId, false, false, false, '', useStereoMusicAudioProfile);
+    await AuthenticateUserStep.executeStep(this, session, attendeeId, false, false, false, '', useStereoMusicAudioProfile);
     await UserAuthenticationCheck.executeStep(this, session);
     await JoinMeetingStep.executeStep(this, session);
-    await UserJoinedMeetingCheck.executeStep(this, session, this.attendeeId);
-    await WaitForRemoteParticipantsToJoinMeeting.executeStep(this, session);
-    await RosterCheck.executeStep(this, session, 2);
-
-    await PlayRandomToneStep.executeStep(this, session, generateStereoTones);
-    // Wait for other participant to start audio
-    await WaitForRemoteParticipantsToTurnAudioOn.executeStep(this, session);
-    // test audio
-    await RemoteAudioCheck.executeStep(this, session, "AUDIO_ON", generateStereoTones);
-    // wait for other participant to finish audio check to finish
-    await WaitForRemoteAudioCheckToComplete.executeStep(this, session);
-    // mute
-    await ClickMicrophoneButton.executeStep(this, session, 'OFF');
-    // wait for other participant to mute
-    await WaitForRemoteParticipantsToTurnAudioOff.executeStep(this, session);
-    // Test Mute
-    await RemoteAudioCheck.executeStep(this, session, "AUDIO_OFF", generateStereoTones);
-    // wait for other participant to finish audio check to finish
-    await WaitForRemoteAudioCheckToComplete.executeStep(this, session);
-
-    await this.waitAllSteps();
+    await UserJoinedMeetingCheck.executeStep(this, session, attendeeId);
```

Comment:
You can just use this one https://github.com/aws/amazon-chime-sdk-js/blob/main/integration/js/utils/SdkTestUtils.js#L5 for both the audio and video tests.

Reply:
I tired to use that one, but I found it is for the very basic join. To use that one, I need to refactor it and change all other tests that call this method. It's not really related to this PR.

Audio and video tests requires some extra configuration, e.g., use simulcast or not and use stereo audio or not.